packages/xod-pm/src/utils.js

```diff
@@ -51,6 +57,15 @@ export const parseLibQuery = R.compose(
   R.match(/^([^@/]+?)\/([^@/]+?)(?:@([^@/]+?))?$/)
 );
 
+// :: String -> LibName
+// It parses libQuery ("xod/core" or "xod/core@0.11.0")
+// and returns lib name without version ("xod/core").
+export const getPureLibName = libQuery => R.compose(
+  qp => `${qp.owner}/${qp.name}`,
+  explodeMaybe(`Expected correct library name format, like "xod/core@0.11.0", but get "${libQuery}"`),
```

Comment:
“but get” → “got”.

Are you sure the pure is the best name here? I think about functional purity. Maybe something like `stripVersionFromLibName` is better?